src/View/Helper/FlashHelper.php

```diff
@@ -67,27 +66,16 @@ class FlashHelper extends Helper
      *    Supports the 'params', and 'element' keys that are used in the helper.
      * @return string|null Rendered flash message or null if flash key does not exist
      *   in session.
-     * @throws \UnexpectedValueException If value for flash settings key is not an array.
      */
     public function render(string $key = 'flash', array $options = []): ?string
     {
-        $session = $this->_View->getRequest()->getSession();
```

Comment:
Should the component also be updated to use the new interface?

Reply:
See https://github.com/cakephp/cakephp/pull/15157#issuecomment-727640037 :)